xml/System.Activities.Presentation.PropertyEditing/EditModeSwitchButton.xml

```diff
@@ -80,7 +80,7 @@
       <Docs>
         <param name="e">
           <see cref="T:System.Windows.DependencyPropertyChangedEventArgs" /> event arguments.</param>
-        <summary>Checks whether the <see cref="F:System.Activities.Presentation.PropertyEditing.PropertyContainer.OwningPropertyContainerProperty" /> has changed (or gained or lost) its owning <see cref="T:System.Activities.Presentation.PropertyEditing.PropertyContainerEditMode" /> and if that is <see langword="true" />, sets up the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> to be the mode to switch to.</summary>
+        <summary>Checks whether the control has changed (or gained or lost) its owning PropertyContainer and if that is <see langword="true" />, sets up the <see cref="P:System.Activities.Presentation.PropertyEditing.EditModeSwitchButton.TargetEditMode" /> to be the mode to switch to.</summary>
```

Comment:
This is an internal comment, rather than documentation of the public method. The method description in source is "Called when any dependency properties of this control change."